site/content/blogs/apprunner-vpc.ja.md

```diff
@@ -0,0 +1,177 @@
+# AWS Copilot による AWS App Runner の VPC 接続
+
+投稿日: 2022 年 6 月 15 日
+
+**Rafael Mosca**, Associate Solutions Architect, WWPS EMEA Central SA
+
+[AWS Copilot CLI](https://aws.github.io/copilot-cli/ja/) は [2020 年にローンチ](https://aws.amazon.com/jp/blogs/containers/introducing-aws-copilot/)されたツールで、それ以来、開発者は [Amazon Elastic Container Service (Amazon ECS)](https://aws.amazon.com/jp/ecs/)、[AWS Fargate](https://aws.amazon.com/jp/fargate/)、[AWS App Runner](https://aws.amazon.com/jp/apprunner/) 上で Linux や Windows コンテナをビルド、管理、運用するために利用されています。
+
+2022 年 2 月、AWS は [AWS App Runner の VPC サポート](https://aws.amazon.com/jp/blogs/aws/new-for-app-runner-vpc-support/)を発表しました。これにより、[AWS App Runner](https://aws.amazon.com/jp/apprunner/) を利用してデプロイする Web アプリケーションや API は、[Amazon Relational Database Service (RDS)](https://aws.amazon.com/jp/rds/) などのサービスで動作するデータベースや、[Amazon Virtual Private Cloud (Amazon VPC)](https://aws.amazon.com/jp/vpc/) でホストされている [Amazon Elastic Container Service (Amazon ECS)](https://aws.amazon.com/jp/ecs/)、[Amazon Elastic Kubernetes Service (EKS)](https://aws.amazon.com/jp/eks/)、[Amazon Elastic Compute Cloud (Amazon EC2)](https://aws.amazon.com/jp/ec2/) で動作する、他のアプリケーションと通信できるようになりました。
+
+AWS App Runner の管理コンソールにて[ドキュメントに記載されている手順](https://docs.aws.amazon.com/ja_jp/apprunner/latest/dg/network-vpc.html)に従って VPC アクセスを有効化することができますが、この手順書では、AWS Copilot を使用して VPC アクセスを有効化して [Amazon Aurora](https://aws.amazon.com/jp/rds/aurora/) データベースに接続する簡単な方法を説明します。
+
+## 背景
+
+[AWS App Runner](https://aws.amazon.com/jp/apprunner/) は、開発者が Web サーバーや API サーバーなどの HTTP ベースのアプリケーションを簡単に実行できるように設計されたフルマネージドサービスです。事前のインフラや経験は必要ありません。ソースコードやコンテナイメージを与えると、App Runner がアプリケーションコンテナをビルドして AWS クラウドにデプロイし、バックグラウンドでリクエストを自動的にスケーリングしてロードバランシングを行います。表示されるのは、HTTPS リクエストを実行できる Service URL だけです。
+![App Runner](./../assets/images/apprunner-vpc-blog/apprunner-arch.png)
+
+Service を作成すると、裏では App Runner が所属する VPC で [Amazon Elastic Container Service (ECS)](https://aws.amazon.com/jp/ecs/) によってオーケストレーションされた [AWS Fargate](https://docs.aws.amazon.com/ja_jp/AmazonECS/latest/userguide/what-is-fargate.html) タスクとして Application コンテナがデプロイされます。デフォルトでは、Application によって開始されたすべてのアウトバウンドトラフィックは、NAT ゲートウェイとこの App Runner が所有する VPC にプロビジョニングされたインターネットゲートウェイを経由してインターネットにルーティングされます。従ってこのモードでは、App Runner でホストされている Application は、インターネット上のパブリックエンドポイントにのみ接続できますが、VPC 内のプライベートエンドポイントに到達することはできません。
```

Comment:
I think we can translate Service, Application into "サービス", "アプリケーション" because the terms Service and Application here is not Copilot Service or Copilot Application.